README.rst

```diff
@@ -59,6 +59,13 @@ Links of interest
 
 * `Donations <https://vdirsyncer.pimutils.org/en/stable/donations.html>`_
 
+Dockerized
+=================
```

Comment:
Can you add a note here mentioning that this is not official? Mostly, I want it to be clear that this is not something maintained upstream. I'm okay with merging this otherwise.

Sorry for the huge delay, I looked into this a bunch of times but always had doubts on how it worked. I see that now you're using cron and a much simpler/unix-like approach to things, which I can actually understand how it works easily.

Reply:
Hey,
yes that's not a problem, i'll add a note, that this container is not official.

Should i create a new PR with the changes?

Reply:
Yes please.

Reply:
I created a new PR with a note, that the Docker container is not an official one
PR #1119